.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+max-line-length: 90
+extend-ignore =
+    # unexpected indentation (comment)
+    E116
+    # continuation line missing indentation or outdented
+    E122
+    # from import
+    E125,E127,E128,E129,E201,E221,E225,E231,E261,E266,E301,E302,E303,E305,E501,E722,F403,F405
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/SETI/rms-opus/pull/1378#issuecomment-2276940859>#1c82e7</a><span> - 08/09/2024, 12:57 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'extend-ignore' list includes 'E722', which ignores the rule against using bare 'except' clauses. This can lead to catching unexpected exceptions and make debugging difficult.
</div>
<div id="fix">
<b>Fix:</b> Remove 'E722' from the 'extend-ignore' list to ensure that bare 'except' clauses are not used.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -9,7 +9,7 @@
      # from import
      E125,E127,E128,E129,E201,E221,E225,E231,E261,E266,E301,E302,E303,E305,E501,E722,F403,F405
      # from application
 -    E125,E127,E128,E129,E201,E221,E225,E231,E261,E266,E301,E302,E303,E501,E722
 +    E125,E127,E128,E129,E201,E221,E225,E231,E261,E266,E301,E302,E303,E501
      # from lib
      E222
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged